### PR TITLE
Add bookings_tag_id param to bookings endpoint

### PR DESCRIPTION
**Pull request by StoneFrog**

### CODE REVIEW

File: content/reference/endpoints/bookings.md [hunk 1]
```diff
@@ -234,7 +236,7 @@ POST /rentals/:rental_id/bookings
 
 <%= render 'json_response', endpoint: "bookings", request: "create",
   scopes: [
-    { bookings_write_owned: "public-bookings_write_owned" },
+    { bookings_write_owned: "bookings_read-bookings_write" },
```
Comment (Azdaroth) [inline content/reference/endpoints/bookings.md]: Why this change?

Comment (StoneFrog) [reply content/reference/endpoints/bookings.md]: It made no sense before. If it has bookings_write_owned scope when you create a booking it returns booking created by app, so it's not limiting attributes, no?

Comment (Azdaroth) [reply content/reference/endpoints/bookings.md]: True.
